docs/troubleshooting.md

```diff
@@ -0,0 +1,67 @@
+# Frequently Asked Questions
```

Comment:
can we isolate FAQs from troubleshooting?

Reply:
Actually this is not FAQ. We would need a better title.

Reply:
Let me rename the file as `faq.md`